<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonelsMed</class>
 <widget class="QDialog" name="PersonelsMed">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MED++</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1281</width>
     <height>721</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Ajouter Personel</string>
    </attribute>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1281</width>
       <height>701</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget{background-image: url(:/AjoutPerso.png);}</string>
     </property>
     <widget class="QLineEdit" name="Occupation">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>229</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Nom">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>310</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Date">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>400</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit#Date
{
color : #ccc
}</string>
      </property>
      <property name="text">
       <string>JJ-MM-AAAA</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="carteid">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>490</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Horaire">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>660</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Salaire">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>570</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Special">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>230</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Prenom">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>310</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Sexe">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>400</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="Tel">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>490</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="LineEdit_6">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>570</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="Mail">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>660</y>
        <width>211</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>1130</x>
        <y>399</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <widget class="QPushButton" name="save">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>-8</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-style:solid;

background-color:#AARRGGBB ;
color:#fff;
border-radius:7px;
}
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_3" native="true">
      <property name="geometry">
       <rect>
        <x>1159</x>
        <y>80</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <widget class="QPushButton" name="retour">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-style:solid;

background-color:#AARRGGBB ;
color:#fff;
border-radius:7px;
}

</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Modifier Personels</string>
    </attribute>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>221</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>80</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Supprimer Personels</string>
    </attribute>
    <widget class="QWidget" name="widget_7" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1281</width>
       <height>691</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget_7
{
	background-image: url(:/SuprimerPerso.png);
}</string>
     </property>
     <widget class="QLineEdit" name="CINsup">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>320</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="widget_8" native="true">
      <property name="geometry">
       <rect>
        <x>1159</x>
        <y>80</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-style:solid;

background-color:#AARRGGBB ;
color:#fff;
border-radius:7px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_9" native="true">
      <property name="geometry">
       <rect>
        <x>1140</x>
        <y>650</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <widget class="QPushButton" name="Delete">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>2</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-style:solid;

background-color:#AARRGGBB ;
color:#fff;
border-radius:7px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Afficher Personels</string>
    </attribute>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1281</width>
       <height>711</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widget_4
{
	background-image: url(:/AfficherPers.png);
}</string>
     </property>
     <widget class="QTableView" name="TabPers">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>240</y>
        <width>1281</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="widget_5" native="true">
      <property name="geometry">
       <rect>
        <x>1150</x>
        <y>79</y>
        <width>120</width>
        <height>41</height>
       </rect>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>2</y>
         <width>111</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
border-style:solid;

background-color:#AARRGGBB ;
color:#fff;
border-radius:7px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
